main.py

```diff
@@ -55,6 +57,10 @@ def run_alexa():
     else:
         talk('Please say the command again.')
 
-
+# In the program in some devices where mic system is bad unboundlocalerror happens and stops the program
+# So rather than stopping the program we are just printing it for our confirmation
 while True:
-    run_alexa()
+    try:
+        run_alexa()
+    except UnboundLocalError:
+        print("Can't give results right now")

```

Comment:
Instead "Can't give results right now" Add the main reason.
And break the loop too😉
```py
print("No commands detected. Alexa has stopped working)
break
#added break statement so it'll not be recursive
```

Reply:
Will add it soon!